recipes/ffx/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```

Comment:
Same thing as above. You can remove the `bld.bat` and `build.sh`, and then add the `script` key line in the `meta.yaml` file `build` sections.